src/api/procedures/removeSecondaryAccounts.ts

```diff
@@ -55,6 +55,7 @@ export async function prepareRemoveSecondaryAccounts(
 export const removeSecondaryAccounts = (): Procedure<RemoveSecondaryAccountsParams> =>
   new Procedure(prepareRemoveSecondaryAccounts, {
     permissions: {
+      // TODO: might not need this -> chain checks if called by primary key, checks if secondary key exists and can be unlinked, then unlinks
```

Comment:
> chain checks if called by primary key,

I don't get this. Technically we don't need to check any permissions since the chain always checks them, but we do this to avoid having the chain catch obvious errors.